daemon/containerd/image_list.go

```diff
@@ -253,9 +256,14 @@ type imageFilterFunc func(image images.Image) bool
 
 // setupFilters constructs an imageFilterFunc from the given imageFilters.
 //
+// containerdListFilters is a slice of filters which should be passed to ImageService.List()
+// filterFunc is a function that checks whether given image matches the filters.
 // TODO(thaJeztah): reimplement filters using containerd filters: see https://github.com/moby/moby/issues/43845
```

Comment:
Is this comment still needed?

Reply:
We might not be able to reimplement Config-reading filters with c8d filter right now, but maybe in future we could work out something that would allow us to perform more filtering on the containerd side.
So while the TODO itself is not immediately actionable, it still mentions an existing issue.
But yeah, at least It could be rephrased a bit.